### PR TITLE
Draft roxygen 6.1.0 release post.

### PR DESCRIPTION
**Pull request by batpigandme**
* _n.b._ date set to 2018-07-30, since it doesn't look like it's actually up on CRAN yet.

### CODE REVIEW

File: content/articles/2018-07-roxygen2-6-1-0.Rmarkdown [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  url: https://unsplash.com/photos/3KEJMfoHoMY
+  author: Graeme Nicholl
+---
+
+```{r setup, include = FALSE}
+library(roxygen2)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We're pleased to announce that [roxygen2](https://CRAN.R-project.org/package=roxygen2)
```
Comment (hadley) [inline content/articles/2018-07-roxygen2-6-1-0.Rmarkdown]: Did you use the random adjective script?

File: content/articles/2018-07-roxygen2-6-1-0.Rmarkdown [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+## New features
+
+The `NAMESPACE` roclet now works in two passes - it first generates the 
+`NAMESPACE` containing only import directives because this can be generated 
+without evaluating the code in the package. This alleviates a problem where it 
+was previously possible to get into a state that you could only get out of by 
+carefully editing the `NAMESPACE` by hand.
+
+`roxygenise()` now uses `pkgload::load_all()` instead of a homegrown solution
```
Comment (hadley) [inline content/articles/2018-07-roxygen2-6-1-0.Rmarkdown]: "This is one of the benefits of the devtools conscious uncoupling" + link to that section on devtools readme/homepage

File: content/articles/2018-07-roxygen2-6-1-0.Rmarkdown [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+was previously possible to get into a state that you could only get out of by 
+carefully editing the `NAMESPACE` by hand.
+
+`roxygenise()` now uses `pkgload::load_all()` instead of a homegrown solution 
+to simulate package loading. This is needed because roxygen2 uses run-time 
+information to generate the documentation. This should reduce S4-related 
+problems, and ensure that `devtools::document()` and `roxygenise()` always have 
+the same behaviour.
+
+`@evalRd` `foo()` evaluates `foo()` defined in the package namespace and
```
Comment (hadley) [inline content/articles/2018-07-roxygen2-6-1-0.Rmarkdown]: This provides a flexible way to extract out duplicated documentation. For example:

```R
today <- function() {
  paste0("Today is \\strong{", format(Sys.Date(), "%A"), "}")
}

#' @evalRd today()
NULL
```

`today()` will be executed every time that you run roxygen2.

There are two small...

File: content/articles/2018-07-roxygen2-6-1-0.Rmarkdown [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+functions in your package.
+
+## Improvements to markdown parsing
+
+Starting from version 6.0.0, roxygen supports markdown markup within most 
+roxygen tags (learn more [here](https://cran.r-project.org/web/packages/roxygen2/vignettes/markdown.html)). You can switch to markdown documentation with
+[`usethis::use_roxygen_md()`](http://usethis.r-lib.org/reference/use_roxygen_md.html), 
+which uses the [roxygen2md](https://github.com/r-lib/roxygen2md) package to 
+convert Rd expressions to markdown. 
+
```
Comment (hadley) [inline content/articles/2018-07-roxygen2-6-1-0.Rmarkdown]: In this version, we've made a number of bug fixes to markdown translation:

**Comment by batpigandme** [comment]
@hadley let me know when we're good to go, and I'll update the date accordingly…

**Comment by hadley** [comment]
Thanks!

**Pull request merged**